docker/local/buid_image.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
```

Comment:
Need an example how to run it. 
Is `--base-image` the `ubuntu:20.04`?

Reply:
I tested this script with `ubuntu:20.04` but any other image can be passed as well